Framework/HistogramData/inc/MantidHistogramData/VectorOf.h

```diff
@@ -116,7 +116,14 @@ template <class T, class CowType> class VectorOf {
       m_data = Kernel::make_cow<CowType>(std::move(data));
     return *this;
   }
-
+  /// Assigns the contents of the range [first, last]
```

Comment:
`last` is not included, so this should read `[first, last)`, see also http://en.cppreference.com/w/cpp/container/vector/assign.